### PR TITLE
2nd Milestone:  use of components, props and state

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
in this milestone, using JSX and Components, create the UI for the Calculator component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (rloterh) [inline src/App.js:5]: - [ ] Your main component is not _class-based_. Kindly ensure that App.js is a _class-based_ component.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Sediq, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (sediqullahbadakhsh) [reply src/App.js]: ok, noted. thank you

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @sediqullahbadakhsh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great 👌 job making the requested changes 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**